x-pack/docs/en/security/authentication/file-realm.asciidoc

```diff
@@ -9,15 +9,16 @@ IMPORTANT:  As the administrator of the cluster, it is your responsibility to
 ensure the same users are defined on every node in the cluster. The {stack}
 {security-features} do not deliver any mechanism to guarantee this.
 
-The `file` realm is primarily supported to serve as a fallback/recovery realm. It
-is mostly useful in situations where all users locked themselves out of the system
-(no one remembers their username/password). In this type of scenarios, the `file`
-realm is your only way out - you can define a new `admin` user in the `file` realm
-and use it to log in and reset the credentials of all other users.
+The `file` realm is very useful as a fallback/recovery realm, for instance in cases where
```

Comment:
Not mandatory, but I prefer spelling out "/" and splitting this long sentence in two:
```suggestion
The `file` realm is very useful as a fallback or recovery realm. For example, use it in cases where
```